src/maintenance/repositories/mirrors/changing.md

```diff
@@ -0,0 +1,22 @@
+# Changing Mirrors
+
+Copy all repository configuration files from `/usr/share/xbps.d` to
```

Comment:
This is a [magic](http://www.catb.org/jargon/html/M/magic.html) instruction and needs to be rewritten to explain why this is being done.